### PR TITLE
Add changelog

### LINKED ISSUE #134: Start maintaining CHANGELOG.md
document some biggest changes since last version:

* [ ] README handling
* [ ] change of paths on Windows and MacOS

### PR DESCRIPTION
**Pull request by daxpedda**
Fixes #134.

Things we have to do/decide:
* [ ] Fill up all versions or decide that we keep some versions undocumented. I will desperately need some help on that.
* [ ] Draft release notes on [Github releases](https://github.com/dpc/crev/releases)
* [ ] Do we want to keep changelogs for all the other projects besides `cargo-crev`?

### CODE REVIEW

**Comment by dpc** [comment]
@Dylan-DPC wanted to take care of relnotes & CHANGELOG. I personally like CHANGELOG, but it's up to him. :)

**Comment by daxpedda** [comment]
Got it, closing it then.

**Comment by dpc** [comment]
@Dylan-DPC So, I was thinking, and I really prefer a `CHANGELOG.md`.

First - I want everything to also be almost release ready, and if I want to release something, I don't want to be blocked on anything etc.

Second - there are multiple project, and it's going to be hard to `git log` and filter which notes are for what.

**Comment by dpc** [comment]
@daxpedda I think we should keep 2 `CHANGELOG.md` files: one for `crev-lib` and one for `cargo-crev`. We will have to repeat some information between them, but oh well.

File: cargo-crev/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Windows cache folder changed from `%AppData%\Local\Dawid Ci,281,,380,arkiewicz\crev\`
+to `%AppData%\Local\crev`.
+- Windows config folder changed from `%AppData%\Roaming\Dawid Ci,281,,380,arkiewicz\crev\`
+to `%AppData%\Roaming\crev`.
+- MacOS cache folder changed from `$HOME/Library/Caches/crev/`
+to `$HOME/Library/Caches/dev.crev/`.
+- MacOS config folder changed from `$HOME/Library/Application Support/crev/`
+to `$HOME/Library/Preferences/dev.crev/`.
+
+## [0.3.0](https://github.com/dpc/crev/compare/cargo-crev-v0.2.0...v0.3.0) - 2018-12-28
```
Comment (dpc) [inline cargo-crev/CHANGELOG.md]: Just ignore previous releases. No need for relnotes for them.

**Comment by dpc** [comment]
I have went ahead with this, and filled it in. Feel free to improve it if you like!

**Pull request merged**